src/libs/ReportUtils.ts

```diff
@@ -3266,7 +3266,7 @@ function getReportName(report: OnyxEntry<Report>, policy: OnyxEntry<Policy> = nu
     }
 
     if (formattedName) {
-        return formattedName;
+        return formatReportLastMessageText(formattedName);
```

Comment:
I am not sure if it is necessary or not, could you explain why we need to update this?

Reply:
@DylanDylann  The update here is to prevent similar bugs in the future if the formattedName variable ever contains a line break.